internal/trainings/trainings.go

```diff
@@ -1,13 +1,63 @@
 package trainings
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	personaldata "github.com/Yandex-Practicum/tracker/internal/personaldata"
```

Comment:
Так как ты не меняешь название пакета, то смысла нет
```suggestion
	"github.com/Yandex-Practicum/tracker/internal/personaldata"
```